x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.test.tsx

```diff
@@ -404,3 +404,149 @@ describe('alerts_list with show only capability', () => {
     // TODO: check delete button
   });
 });
+
+describe('refresh button', () => {
```

Comment:
Instead of adding a new `describe` block with its own setup (which adds time to the tests), let's add the test to an existing unit test.

I would suggest adding a test to the `alerts_list component with items` tests above:

Inside `it('renders table of alerts'`, I would add 
```
expect(wrapper.find('[data-test-subj="refreshAlertsButton"]').exists()).toBeTruthy();
```

and then add another test to make sure the `loadAlerts` function is being called when the refresh button is clicked

```
it('loads alerts when refresh button is clicked', async () => {
    await setup();
    wrapper.find('[data-test-subj="refreshAlertsButton"]').first().simulate('click');
    expect(loadAlerts).toHaveBeenCalled();
});
```